ghcide/test/exe/Main.hs

```diff
@@ -3954,7 +3954,7 @@ findDefinitionAndHoverTests = let
   chrL36 = Position 41 24  ;  litC   = [ExpectHoverText ["'f'"]]
   txtL8  = Position 12 14  ;  litT   = [ExpectHoverText ["\"dfgy\""]]
   lstL43 = Position 47 12  ;  litL   = [ExpectHoverText ["[8391 :: Int, 6268]"]]
-  outL45 = Position 49  3  ;  outSig = [ExpectHoverText ["outer", "Bool"], mkR 46 0 46 5]
+  outL45 = Position 49  3  ;  outSig = [ExpectHoverText ["outer", "Bool"], mkR 50 0 50 5]
```

Comment:
Sorry, I don't know this test, do you change it because the test did not check what we thought it did, or would it make sense to have both test lines?

Reply:
hi, I revised the linked issue manually and I checked hover on top level type signatures works, but no inner ones

But the test still was failing cause the position was different. Correcting the position it passes

Reply:
Awesome, nice!